pages/common/pdf/client/reducers/index.js

```diff
@@ -0,0 +1,12 @@
+import { combineReducers } from 'redux';
+import pil from './pil';
+import licenceHolder from './profile';
+import establishment from './establishment';
+
+const rootReducer = combineReducers({
+  pil,
+  licenceHolder,
+  establishment
+});
+
+export default rootReducer;
```

Comment:
These reducers don't really do anything, but if they don't exist, `createStore()` starts throwing warnings in the log? I've kept them for now to get it working but I'm expecting we'll remove / replace them.

Reply:
the app needs state, which it gets from the redux store, which this rootReducer creates

Reply:
however, as it's all static, you don't need 3 reducers to do one thing, just have one and call it static or something?

Reply:
all this is doing is creating a static state object that looks like this: 
```js
{
  pil: {},
  establishment: {},
  ...
}
```

Reply:
actually, thinking about this, it doesn't need redux at all, all of the store stuff can go, this is 100% new code so doesn't have to follow the same pattern as our complex application - you could just pass in props to the <App /> on init